pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -658,30 +659,52 @@ type InterfaceTarget struct {
 	Managed string `xml:"managed,attr,omitempty"`
 }
 
+// Tentatively keep "Name" exported for if we privatize the field, "kubectl get vm" does not output IP.
 type Alias struct {
-	Name string `xml:"name,attr"`
+	Name        string
+	userDefined bool
 }
 
-type UserAlias Alias
+// Package private, responsible to interact with xml.Encoder and xml.Decoder
+type userAliasXML struct {
+	Name string `xml:"name,attr"`
+}
 
 type Rom struct {
 	Enabled string `xml:"enabled,attr"`
 }
 
+func NewAlias(aliasName string) *Alias {
```

Comment:
Maybe name this `NewLibvirtManagedAlias` or something and add docs on the constructors.

Reply:
Changed the func name.